smoke_tests/processorFunctions.js

```diff
@@ -0,0 +1,104 @@
+const AWS = require('aws-sdk');
+const request = require('request');
+
+module.exports = {
+  addCredentials: addCredentials,
+};
+
+const stageSecrets = {
+  auth0Url: 'https://wellcomecollection-stage.eu.auth0.com/oauth/token',
+  apiKeySecretId: 'identity/stage/account_management_system/api_key',
+  credentialsSecretId: 'identity/stage/smoke_test/credentials',
+};
+
+const prodSecrets = {
+  auth0Url: 'https://wellcomecollection.eu.auth0.com/oauth/token',
+  apiKeySecretId: 'identity/prod/account_management_system/api_key',
+  credentialsSecretId: 'identity/prod/smoke_test/credentials',
+};
+
+const secretsManager = new AWS.SecretsManager();
+
+let apiKey;
```

Comment:
When does this get set?

Reply:
This is some vestigial caching stuff for when we make multiple requests - I should make it work again.